sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_enums.py

```diff
@@ -39,6 +39,13 @@ class KeyOperation(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
     export = "export"
 
 
+class KeyRotationPolicyAction(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
```

Comment:
What is CaseInsensitiveEnumMeta? I'm just curious if it does what I think it does 😄

Reply:
[Here's](https://docs.microsoft.com/en-us/python/api/azure-core/azure.core.caseinsensitiveenummeta?view=azure-python) the class reference -- having our enums inherit from this is a recent addition to the Python guidelines